pkg/apiserver/apiserver.go

```diff
@@ -162,16 +170,23 @@ func NewCdiAPIServer(bindAddress string,
 
 	})
 
-	validatingwebhook.SetClient(client)
+	err = app.createValidatingWebhook()
+	if err != nil {
+		return nil, errors.Errorf("failed to create validating webhook: %s", err)
```

Comment:
Should it be a %v for an err, or err.Error() if you use %s?

Reply:
%s is okay for error type but not all types